docs/commands/rhoas_cluster_connect.md

```diff
@@ -6,7 +6,7 @@ Connect your services to Kubernetes or OpenShift
 
 Connect your application services to an Kubernetes or OpenShift cluster.
 
-Note: Before you can connect your application service to OpenShift, you need to install the RHOAS Operator in your OpenShift cluster. For installation instructions, see https://github.com/redhat-developer/app-services-guides/tree/main/service-discovery#installing-the-rhoas-operator-on-openshift.
+Note: Before you can connect your application service to OpenShift, you need to install the RHOAS Operator in your OpenShift cluster. For installation instructions, see https://github.com/redhat-developer/app-services-guides/tree/main/docs/kafka/service-binding-kafka#installing-the-rhoas-operator-on-openshift.
```

Comment:
Suggest "you must install the RHOAS Operator..." as recommended by IBM Style Guide

Reply:
The rhoas_cluster_connect.md file will be updated when the update is made in the cluster.en.toml doc and the command docs are regenerated, but as mentioned below, that update is outside the scope of this PR.